src/components/verification-grid/verification-grid.ts

```diff
@@ -198,20 +218,16 @@ export class VerificationGridComponent extends AbstractComponent(LitElement) {
   private showingSelectionShortcuts = false;
   private selectionHead: number | null = null;
   private doneRenderBoxInit = false;
+  private isOverlapping = signal<boolean>(false);
```

Comment:
so there's one signal bound to all tiles?

so the assumption here is that all tiles have uniform content and all their interaction observers fire at the same time and report the same thing - because if they don't, the values are just clobbered based on who reports last

Reply:
We talked about this offline, and I have made the appropriate fix